Jenkinsfile

```diff
@@ -187,7 +187,7 @@ VERSION=${env.VERSION}-SNAPSHOT""")
       // Required to enable the flaky test reporting with GitHub. Workspace exists since the post/always runs earlier
       dir("${BASE_DIR}"){
         notifyBuildResult(prComment: true,
-                          slackComment: true, slackNotify: (isBranch() || isTag()),
+                          slackComment: true,
```

Comment:
Default behaviour honours builds for branches/tags which failed.

https://github.com/elastic/apm-pipeline-library/blob/53d5532084be9f9ea4e6425bc3fe165fa6e9ab90/vars/notifyBuildResult.groovy#L65